Tools/Release/update_dependencies.py

```diff
@@ -42,6 +42,22 @@ def update(args):
         repo_dict["picsar"]["tags"] = (
             "https://api.github.com/repos/ECP-WarpX/picsar/tags"
         )
+    if args.all or args.pybind11:
+        repo_dict["pybind11"] = {}
+        repo_dict["pybind11"]["commit"] = (
+            "https://api.github.com/repos/pybind/pybind11/commits/master"
+        )
+        repo_dict["pybind11"]["tags"] = (
+            "https://api.github.com/repos/pybind/pybind11/tags"
+        )
```

Comment:
Note that we do not do a weekly tracking of pybind11. We manually update when we want a newer release.

Most importantly, when we release WarpX we do _not_ depend on the pybind11 development branch (or any dependency as of their development branch).